### PR TITLE
Add link to TSC governance page to quick link list

### PR DESCRIPTION
**Pull request by smoyth**

### CODE REVIEW

File: TSC1/README.md [hunk 1]
```diff
@@ -11,7 +11,9 @@ The Technical Steering Committee is the technical governing body for Open Data K
 - [TSC Forum Category](https://forum.opendatakit.org/c/governance/tsc)
 - [TSC Forum Tag](https://forum.opendatakit.org/tags/tsc)
 - ODK Project Pages: [Briefcase](https://github.com/opendatakit/briefcase), [Build](https://github.com/opendatakit/build), [Collect](https://github.com/opendatakit/collect), [Documentation](https://github.com/opendatakit/docs), [JavaRosa](https://github.com/opendatakit/javarosa), [ODK XForms specification](https://github.com/opendatakit/xforms-spec), [Validate](https://github.com/opendatakit/validate), [XLSForm offline](https://github.com/opendatakit/xlsform-offline), [XLSForm online](https://github.com/opendatakit/xlsform-online)
+- [TSC Governance Guidelines](https://github.com/lognaturel/governance/blob/master/TECHNICAL-STEERING-COMMITTEE.md)
```
Comment (yanokwa) [inline TSC1/README.md]: s/lognaturel/opendatakit

**Comment by yanokwa** [comment]
@smoyth Thanks! Next time, consider [squashing the commits](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html) since the commit history here isn't super useful and do a `git push -f` to force the PR to update.

**Comment by smoyth** [comment]
@yanokwa Can't you squash them on merge?

**Comment by yanokwa** [comment]
Yes, but my stance is if a contributor can do it, a reviewer shouldn't do it. And you might want to say squash 2 of the commits and keep one, which I can't do easily in the GH interface. TLDR: I think we should push as much work to the contributor because reviewers have enough to do.

**Comment by adamvert** [comment]
I agree with both of you :) Which is why we get contributors to actually press the merge button once a PR has been accepted. Usually they squash everything, but if they want to do something more exotic, that's fine too.

**Pull request merged**